performance-test/support-files/location_code.txt

```diff
@@ -0,0 +1,40 @@
+locationCode
```

Comment:
Can this location code, not be part of hierarchy_level ? Thought process is to keep the number of support-files to a till and club the related files together

Reply:
Have made the location code part of the hierarchy level file.